### PR TITLE
[esw, teleop] 2022 ERD/ES code

### PR DESCRIPTION
**Pull request by cgiger00**
merging in code for mock ERD/ES mission

### CODE REVIEW

**Review by CameronTressler — APPROVED** [review]

File: base_station/gui/src/components/ERDTask.vue [hunk 1]
```diff
@@ -168,12 +154,17 @@ export default {
       // Subscriptions
       [
         {'topic': '/odometry', 'type': 'Odometry'},
-        {'topic': '/ra_position', 'type': 'RAPosition'},
+        {'topic': '/sensors', 'type': 'Sensors'},
+        {'topic': '/temperature', 'type': 'Temperature'},
+        {'topic': '/kill_switch', 'type': 'KillSwitch'},
```
Comment (tabiosg) [inline base_station/gui/src/components/ERDTask.vue:159]: yo yo yo what's the kill switch doing here

File: base_station/gui/src/components/ERDTask.vue [hunk 2]
```diff
@@ -168,12 +154,17 @@ export default {
...
+        {'topic': '/camera_servos', 'type': 'CameraServos'},
```
Comment (tabiosg) [inline base_station/gui/src/components/ERDTask.vue:160]: nooooo don't bring these back!

File: base_station/gui/src/components/LCMEcho.vue [hunk 3]
```diff
@@ -48,13 +48,12 @@
           lcm: false
         },
         messages: [],
-        viewing: {
+        viewing: { // TODO: alphbatize
```
Comment (tabiosg) [inline base_station/gui/src/components/LCMEcho.vue:51]: TODO: SPELLING

alphabetize

File: base_station/gui/src/components/PDBData.vue [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  components: { SpectralData },
+    data(){
+        return{
+            currentDraw:0,
+            temperature:0,
+            activeColor:"red"
+        }
+    },
+    created: function(){
+        setTimeout(() => {  this.currentDraw = 36.6;
```
Comment (tabiosg) [inline base_station/gui/src/components/PDBData.vue:22]: idk what this but is this from the mock values

File: base_station/gui/src/static/rover_msgs.json [hunk 5]
```diff
@@ -1203,19 +1233,18 @@
...
         "RGBFrame",
         "RGBLED",
         "SAMotors",
         "SAPosition",
+        "ScoopLimitSwitchEnabled",
         "SensorPackage",
         "Sensors",
         "Servo",
-        "ServoCmd",
+        "ServoCmds",
```
Comment (tabiosg) [inline base_station/gui/src/static/rover_msgs.json:1247]: WHAT'S THIS? THE LCM? the lcm is ServoCmd

**Comment by tabiosg** [comment]
i will approve after i test with arm first

File: base_station/gui/src/components/ERDTask.vue (hunk 1 above)
Comment (CameronTressler) [reply base_station/gui/src/components/ERDTask.vue:159]: ya

File: base_station/gui/src/components/ERDTask.vue (hunk 2 above)
Comment (CameronTressler) [reply base_station/gui/src/components/ERDTask.vue:160]: ya

File: base_station/gui/src/components/LCMEcho.vue (hunk 3 above)
Comment (CameronTressler) [reply base_station/gui/src/components/LCMEcho.vue:51]: ya

File: base_station/gui/src/components/PDBData.vue (hunk 4 above)
Comment (CameronTressler) [reply base_station/gui/src/components/PDBData.vue:22]: We don't use this file anymore, should be using PDBFuse.vue

File: base_station/gui/src/static/rover_msgs.json (hunk 5 above)
Comment (CameronTressler) [reply base_station/gui/src/static/rover_msgs.json:1247]: This part of rover msgs is high key super unnecessary anyways

File: base_station/gui/src/components/ERDTask.vue (hunk 1 above)
Comment (cgiger00) [reply base_station/gui/src/components/ERDTask.vue]: lol

File: base_station/gui/src/components/PDBData.vue (hunk 4 above)
Comment (cgiger00) [reply base_station/gui/src/components/PDBData.vue]: probs should take that out..

File: base_station/gui/src/static/rover_msgs.json (hunk 5 above)
Comment (cgiger00) [reply base_station/gui/src/static/rover_msgs.json]: can you get rid of Servo while youre at it

File: base_station/cameras/README.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+Code to control the USB Cameras
```
Comment (tabiosg) [inline base_station/cameras/README.md]: I don't know how this is here? On my computer I deleted all of base_station/cameras...

Comment (tabiosg) [reply base_station/cameras/README.md]: ok seems like git is being weird... the folder doesn't actually exist i think

Comment (CameronTressler) [reply base_station/cameras/README.md]: Yeah I feel like I've removed things multiple times at this point...

Comment (CameronTressler) [reply base_station/cameras/README.md]: I could be imagining it though

**Review by tabiosg — APPROVED** [review]

**Review by CameronTressler — APPROVED** [review]

**Pull request merged**